internal/collector/lt_pg17_fast_metrics.yaml

```diff
@@ -4,33 +4,22 @@
 # https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/-/receiver/sqlqueryreceiver#metrics-queries
 # https://github.com/CrunchyData/pgmonitor/blob/development/sql_exporter/common/crunchy_global_collector.yml
 
-  - sql: >
-      SELECT c.buffers_checkpoint AS buffers_written
-      FROM pg_catalog.pg_stat_bgwriter c;
-    metrics:
-      - metric_name: ccp_stat_bgwriter_buffers_checkpoint
```

Comment:
When did the ccp_stat_bgwriter_* metrics go away?

Reply:
Not all `ccp_stat_bgwriter_*` metrics went away, but in pgMonitor v5.1.0 I think, ones that were "checkpoint" related moved to `ccp_stat_checkpointer` and then in pgmonitor-extension, the function checks if the PG version is gte 17, and if so, it uses `pg_stat_checkpointer` and if it's less than 17 it uses `pg_stat_bgwriter`.